backend/local/local.go

```diff
@@ -265,6 +167,10 @@ func (b *Local) Create() (backend.Blob, error) {
 
 // Construct path for given Type and name.
 func filename(base string, t backend.Type, name string) string {
+	if t == backend.Config {
```

Comment:
why is this a special case here and not implemented in the `dirname` function like the other types?